pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -22,14 +22,16 @@ func ResolveContainerEnv(client client.Client, logger logr.Logger, podSpec *core
 
 	var container corev1.Container
 	if containerName != "" {
+		findMathContainer := false
```

Comment:
there's probably typo in the variable name, eg. `containerWithNameFound` or something like that

Reply:
fixed. `containerWithNameFound` would be better.